### DISCUSSION TITLE
How to transfer firefly data between machines

### DISCUSSION THREAD

**Discussion by Asteliks** [firefly-iii/firefly-iii#6222] [Firefly III]
This may be rather obvious to some but how can I migrate my old docker based firefly III data to firefly III from [TrueCharts](https://truecharts.org/apps/stable/fireflyiii/) in [TrueNas Scale](https://www.truenas.com/truenas-scale/)?

I have asked over at TrueCharts discord but they couldn't help me.

I wish we could have a simple Firefly transfer tool that moves all the data

**Comment by JC5** [comment]
Ask them how to backup and restore TrueNas Scale installations. It's the same thing.

**Reply by Asteliks** [reply]
at the moment there doesn't seem to be a container level backup. Only way you can backup is through the entire thing
